Makefile

```diff
@@ -168,7 +170,7 @@ release-notes-copy-to-docs: ## Copy release notes into documentation
 
 .PHONY: backend-docker-start
 backend-docker-start: ## Starts a Docker-based backend for development
-	$(MAKE) -C "./packages/volto/" backend-docker-start
+	$(MAKE) -C "./packages/volto/" backend-docker-start -e $(LANGUAGE)
```

Comment:
I think the Volto makefile has to receive this variable:

https://github.com/plone/volto/blob/9e312ed0b1a977bcc42a2e6a7d1b9406ae7fb605/packages/volto/Makefile#L103

Reply:
@wesleybl I made that adjustment and it works for me now if I do `LANGUAGE=pt-br make backend-docker-start`.

(I ran `docker volume rm volto-backend-data` first.)

Reply:
@davisagli Now it worked for me too!